libnavigation-core/src/test/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSessionTest.kt

```diff
@@ -135,6 +136,7 @@ class MapboxDirectionsSessionTest {
         verify(exactly = 1) { observer.onRoutesChanged(routes) }
     }
 
+    @Ignore
```

Comment:
Was this needed because of the Faster route changes introduced as part of this PR? See comment above.